src/main/java/eu/europeana/metis/sandbox/service/workflow/HarvestServiceOaiPmh.java

```diff
@@ -0,0 +1,10 @@
+package eu.europeana.metis.sandbox.service.workflow;
```

Comment:
Why do we have a separate interface/class just dedicated to OAI-PMH harvesting?
The `HarvestService` interface contains all other harvesting methods, why don't we put it all together in one class?
OAI-PMH is not a harvesting method?

Reply:
Will do. All calls will be made HarvestService. 
HarvestServiceOaiPmh and implementation will be removed/refactor